src/lib/retrieveCommunityEvents.ts

```diff
@@ -1,8 +1,5 @@
 import type { CommunityEvent, EventDateParams } from '../types'
 import { retrieveAllCollectionDataOfType } from '$lib/retrieveCollectionData'
-import dayjs from 'dayjs'
```

Comment:
Do we need to remove dayjs from the `package.json` file since we're removing this? Looks like it won't be needed anymore?

Reply:
It's not required to remove it, but certainly cleaner. I did intend to remove them, but forgot in the process.
AFAIK, dayjs won't even appear in the target code. I've removed them now though.